src/python/python-proto/python_proto/__init__.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from abc import ABCMeta, abstractmethod, abstractstaticmethod
+from typing import Any
+
+
+class SerDe(metaclass=ABCMeta):
+    @classmethod
+    def __subclasshook__(cls, subclass: Any) -> bool:
```

Comment:
I believe it would be valid to replace the Any with `SerDe`, and that would work, otherwise you could use a TypeVar.

Reply:
done